solr/solrj/src/test/org/apache/solr/client/solrj/routing/RequestReplicaListTransformerGeneratorTest.java

```diff
@@ -88,6 +88,19 @@ public void replicaTypeAndReplicaBase() {
         )
     );
 
+    // Add a PULL replica so that there's a tie for "last place"
+    replicas.add(
+        new Replica(
+            "node4",
```

Comment:
node5?

Reply:
Ah yes; fixed, thanks for catching that!